doctr/utils/visualization.py

```diff
@@ -182,72 +182,76 @@ def visualize_page(
     if interactive:
         artists: List[patches.Patch] = []  # instantiate an empty list of patches (to be drawn on the page)
 
-    for block in page["blocks"]:
-        if not words_only:
-            rect = create_obj_patch(
-                block["geometry"], page["dimensions"], label="block", color=(0, 1, 0), linewidth=1, **kwargs
-            )
-            # add patch on figure
-            ax.add_patch(rect)
-            if interactive:
-                # add patch to cursor's artists
-                artists.append(rect)
-
-        for line in block["lines"]:
+    if isinstance(page["blocks"], list):
+        page["blocks"] = {"words": page["blocks"]}
+
+    for key, value in page["blocks"].items():
+        for block in value:
             if not words_only:
                 rect = create_obj_patch(
-                    line["geometry"], page["dimensions"], label="line", color=(1, 0, 0), linewidth=1, **kwargs
+                    block["geometry"], page["dimensions"], label=f"block_{key}", linewidth=1, **kwargs
```

Comment:
No more `color=(0, 1, 0)` here ?

Reply:
to make the color change for each different class. I don't know it it's better to have a way to choose color depending on the number of classes

Reply:
Thanks Done and pushed.